### DISCUSSION TITLE
How to style the tiptap editor generated content?

### DISCUSSION THREAD

**Discussion by alokVishu** [awcodes/filament-tiptap-editor#481] [General]
I'm using the tip tap editor plugin. I can access the Grid builder, tables, and many other features in my backend. However, when I use the same code in the front end, it has no styling. Do I need to style them manually, such as the grid or table?

**Comment by awcodes** [comment]
When displaying the content outside of the editor you will need to provide your own styles. The plugin can't make assumptions about the styling of your site. :)

I usually use the Tailwind typography plugin as a good base around the content when I display it. but you will need to add things for grids and tables.
